### ISSUE TITLE
Split into 2 repos (server & client)

### ISSUE THREAD

**Issue by peduarte** [devcon-london/devcon-london.github.io#2]
I'm thinking we should create another repo to abstract the server from the client.
At the moment they're both on the same repo.

I already separated this in order to push the api to heroku...

Any ideas?

**Comment by grudelsud** [comment]
hey

do we need both server and client? seems a bit too complicated to me, not to mention that we'll need to rely on heroku and have another layer of authentication there, which is not very maintainable. would it be possible just to have the frontend that uses [google form API](https://developers.google.com/apps-script/reference/forms/form-app) and submit to the right place?

**Comment by peduarte** [comment]
I can take a look at it... at the moment the server side is scraping the google form and returning it in a json format: https://devconlondon-api.herokuapp.com/api/google-form?formId=1FvacYUr7XxypGawVUOquLRdoH_QNy1wRxiCMZ6SN304

Thats about it...

**Comment by peduarte** [comment]
@grudelsud either the docs is really really bad, or i am misunderstanding it!
but I think that api is not for javascript, it is Apps Script, which is to create Add ons for Google apps stuff...

**Comment by peduarte** [comment]
The API that is currently hosted on Heroku is here: https://github.com/peduarte/Google-Form-API/

So any changes made to the API on this repo won't really do anything on prod env.

For now ¯\_(ツ)_/¯

**Comment by peduarte** [comment]
Update:

I managed to get the server running on Heroku from this repo now.
https://devconlondon.herokuapp.com/api/google-form?formId=1FvacYUr7XxypGawVUOquLRdoH_QNy1wRxiCMZ6SN304

So now the benefit is, server and client are both within the same repo, so easier to manage server stuff.

@grudelsud This means we still need to deploy to two different places, but I believe I've made it as simple as possible, refer to readme.

@grudelsud and @silviopaganini gimme your Heroku usernames and i'll make sure you have deployment access.
